backend/kale/common/podutils.py

```diff
@@ -300,3 +307,63 @@ def compute_component_id(pod):
     component_id = component_name + "@sha256=" + component_spec_digest
     log.info("Computed component ID: %s", component_id)
     return component_id
+
+
+def get_snapshotclasses(label_selector=""):
```

Comment:
If you don't use the `label_selector` in future PRs, then let's remove it. Same thing for `list_snapshotclass_storage_provisioners`

Reply:
Label_selector is used in later logic.